packages/api/src/command/webhook/webhook.ts

```diff
@@ -134,17 +134,9 @@ export const processRequest = async (
     //eslint-disable-next-line @typescript-eslint/no-explicit-any
   } catch (error: any) {
     console.log(`Failed to process WH request: ${errorToString(error)}`);
-    capture.error(error, {
-      extra: {
-        webhookRequestId: webhookRequest.id,
-        webhookUrl,
-        context: `webhook.processRequest`,
-        error,
-      },
-    });
```

Comment:
Important: we're stopping to report WH errors to Sentry - context [1](https://metriport.slack.com/archives/C04DMKE9DME/p1691788872599879) [2](https://metriport.slack.com/archives/C04T256DQPQ/p1699984505555239?thread_ts=1699984449.023109&cid=C04T256DQPQ) [3](https://metriport.slack.com/archives/C04T256DQPQ/p1695906997179609?thread_ts=1695880466.039099&cid=C04T256DQPQ)